### ISSUE TITLE
Could not find a declaration file for module '@vuelidate/core'

### ISSUE THREAD

**Issue by NotBjoggisAtAll** [vuelidate/vuelidate#1189]
Hi, I get the following error after I upgraded a bunch of dependencies on my vue project. I would guess the typescript upgrade from 4 to 5 was one of the reasons this error now appears. 

```
node_modules/@vuelidate/core/dist/index.mjs' implicitly has an 'any' type.

There are types at 'node_modules/@vuelidate/core/index.d.ts', but this result could not be resolved when respecting package.json "exports". The '@vuelidate/core' library may need to update its package.json or typings.
```

It prevents me from running type-checks with: 
`"type-check": "vue-tsc --noEmit -p tsconfig.vitest.json --composite false",`

I have fixed the issue temporarily by declaring a module in a `index.d.ts` file, but I think changes should be made to fix this.
`declare module '@vuelidate/core'`

**Comment by volarname** [comment]
+1

**Comment by charlesvinoth** [comment]
+1

**Comment by cerinoligutom** [comment]
+1

**Comment by apappas1129** [comment]
+1

**Comment by lf-wxp** [comment]
+1

**Comment by j2c-arnuth** [comment]
+1

**Comment by davidkjackson54** [comment]
@NotBjoggisAtAll can you possibly share you workaround index.d.ts till this gets fixed?

**Comment by cerinoligutom** [comment]
@davidkjackson54  Saw this PR https://github.com/vuelidate/vuelidate/pull/1184 and just patched our `node_modules` until fixed upstream. If you're using using `pnpm`, there's [`pnpm patch`](https://pnpm.io/cli/patch), otherwise use [`patch-package`](https://www.npmjs.com/package/patch-package).

**Comment by davidkjackson54** [comment]
Is https://github.com/vuelidate/vuelidate/pull/1184  planned to be promoted at some point in the near future to resolves these issues  -1189 and 1193?

**Comment by callumacrae** [comment]
Can confirm that updating to 2.0.3 fixed this issue!

**Comment by NotBjoggisAtAll** [comment]
Fixed in 2.0.3
